Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -41,6 +41,21 @@ public ApplicationPOMModel()
 
         private string mPageURL = string.Empty;
 
+        PageLoadFlowType mPageLoadFlow;
```

Comment:
please change "PageLoadFlowType" to "ePageLoadFlowType" - all enum types needs to start with "e"

Reply:
renamed